.ci/example_project/MetaTestPkg/Project.toml

```diff
@@ -0,0 +1,7 @@
+name = "MetaTestPkg"
```

Comment:
Is there a reason, this package is differently named compared to the other example packages?

Reply:
Yes, it is documented (the PR is not merged yet): https://github.com/QEDjl-project/IntegrationTests.jl/pull/9/files#diff-8d933ae32a8f6c840044b8224e06f186bbf57afed1ccf71e82e6b0d205640625R105